### PR TITLE
Engine rewrite

### PR DESCRIPTION
**Pull request by TiaSinghania**
Light cleanup of engine.py, made sure noise was actually applied properly

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [X] Optimization
- [ ] Documentation Update

## Description
_Describe the CHANGES, the REASONING, and BENEFITS of this PR._

## Related Tickets & Documents

- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: rb_ws/src/buggy/buggy/util.py
Comment (mehulgoel873) [inline rb_ws/src/buggy/buggy/util.py:1]: Good idea, I feel there should be a better way (like a json file), do you want to look into that?

File: rb_ws/src/buggy/buggy/simulator/engine.py [hunk 1]
```diff
@@ -8,25 +9,20 @@
...
-    WHEELBASE_SC = 1.17
-    WHEELBASE_NAND= 1.17
+
 
     def __init__(self):
         super().__init__('sim_single')
         self.get_logger().info('INITIALIZED.')
 
 
         self.starting_poses = {
```
Comment (mehulgoel873) [inline rb_ws/src/buggy/buggy/simulator/engine.py:23]: I feel these could also be going to constants, this feels kinda sketch ngl

File: rb_ws/src/buggy/buggy/simulator/engine.py [hunk 2]
```diff
@@ -51,43 +44,43 @@ def __init__(self):
         self.steering_subscriber = self.create_subscription(
```
Comment (mehulgoel873) [inline rb_ws/src/buggy/buggy/simulator/engine.py:44]: Issue with self.pose_publisher, that we didn't discuss yet. The INS output now is in ECEF, so either we simulate that or provide clean_states, which are specified as part of the wiki page

Comment (mehulgoel873) [inline rb_ws/src/buggy/buggy/simulator/engine.py:44]: Put all subscribers together, all publishers together.

**Review by mehulgoel873 — CHANGES_REQUESTED** [review]

File: rb_ws/src/buggy/buggy/simulator/engine.py (hunk 1 above)
Comment (TiaSinghania) [reply rb_ws/src/buggy/buggy/simulator/engine.py:23]: (by these are you referring to the start poses?) they are constants, but they are definitely not relevant to any system besides the sim so i left them in here to not clutter constants - easy fix if we want tho

Comment (mehulgoel873) [reply rb_ws/src/buggy/buggy/simulator/engine.py:23]: Let's keep it as is then.

File: rb_ws/src/buggy/buggy/simulator/engine.py (hunk 2 above)
Comment (TiaSinghania) [reply rb_ws/src/buggy/buggy/simulator/engine.py]: published as clean_slate

**Review by mehulgoel873 — APPROVED** [review]

**Pull request merged**